package.json

```diff
@@ -131,15 +131,28 @@
     "router"
   ],
   "release": {
+    "branches": [
+      "+([0-9])?(.{+([0-9]),x}).x",
```

Comment:
We haven't tested this with travis ci, but hopefully it should work

Reply:
@nellyk it is the default of `semantic-release`